rero_ils/modules/documents/views.py

```diff
@@ -597,3 +597,41 @@ def online_holdings(document_pid, viewcode='global'):
         library_holdings.append(record)
         holdings[library['name']] = library_holdings
     return holdings
+
+
+@blueprint.app_template_filter()
+def subjects(subjects, language):
+    """Format the subjects according to the available keys.
+
+    : param subjects: record subjects
+    : param language: current language on interface
+    """
+    data = []
+    for subject in subjects:
```

Comment:
This can be probably refactored using a for loop over a list of subject properties:
```python
for key in ['term', 'preferred_name', 'title']:
    ...
```